log_collection.sh

```diff
@@ -0,0 +1,652 @@
+#!/bin/sh
```

Comment:
We have decided to:
 - send only necessary data (Vipin has list), start small and grow it as needed
 - check if we could use fpaste to upload the data
   - could we maintain our own instance?